src/main/java/com/akademia/educators_back/exception/TextIsToLongOrToShortException.java

```diff
@@ -0,0 +1,7 @@
+package com.akademia.educators_back.exception;
+
+public class TextIsToLongOrToShortException extends RuntimeException{
```

Comment:
bardziej bym to nazwał TextLenghtException, i ewentualnie w komunikacie dać szczegóły